internal/cache/metadata/stat_cache_test.go

```diff
@@ -36,34 +36,22 @@ type invariantsCache struct {
 func (c *invariantsCache) Insert(
```

Comment:
Do we need this invariantsCache now? If not then we can delete this.

Reply:
Yes, this is the wrapper class that we are using for all the unit tests for statCache, so this class is needed. We could rename it though probably.

Reply:
@raj-prince ^

Reply:
Done.